R/shared-methods.R

```diff
@@ -68,7 +68,7 @@ check_country_available <- function(country = character(), level = 1,
 #' @param totals Logical, defaults to FALSE. If TRUE, returns totalled
 #'  data per region up to today's date. If FALSE, returns the full dataset
 #'  stratified by date and region.
-#' @param localise Logical, defaults to TRUE. Should region names be localised.
+#' @param localise Logical, defaults to TRUE. Should region names be localized.
```

Comment:
whom some pesky person has used American English in the function name.... (not sure there is much to be done about this)